admin-dev/themes/new-theme/js/pages/feature-flag/index.js

```diff
@@ -29,11 +29,36 @@ const {$} = window;
 
 $(() => {
   const $submitButton = $('#submit-btn-feature-flag');
+  $submitButton.prop('disabled', true);
   const $form = $('#feature-flag-form');
+  const $formInputs = $('#feature-flag-form input');
+  const initialState = $form.serialize();
+
+  $formInputs.change(() => {
+    if (initialState === $form.serialize()) {
+      $submitButton.prop('disabled', true);
+    } else {
+      $submitButton.prop('disabled', false);
+    }
+  });
 
   $submitButton.on('click', (event) => {
     event.preventDefault();
 
+    const formData = $form.serializeArray();
+    let oneFlagIsEnabled = false;
+
+    if (initialState === $form.serialize()) {
+      return;
+    }
+
+    for (let i = 0; i < formData.length; i += 1) {
+      if ((formData[i].name !== 'form[_token]') && (formData[i].value !== '0')) {
```

Comment:
Mmm, I may be wrong but it seems like you only check the new for content (is there one enabled) But you didn't check the previous state I thought the modal is supposed to be shown only when one input which was off has been enabled

I know you already check for modification to enable the button but in the following example:
- Initial state: buttonA on, button B on
- new state: buttonA on, button B off
=> modal is shown but it shouldn't

Reply:
Very good insight, and you're 100% right you catched a bug

I fixed it by comparing form input content with its initial content https://github.com/PrestaShop/PrestaShop/pull/24052/commits/65ce81f223a539b3ed0b4254b8ed69ec01fc762a